src/hierarchical_deterministic/bip39/bip39_entropy.rs

```diff
@@ -21,7 +21,7 @@ macro_rules! entropy_with_byte_counts {
             $(
                 #[doc = $expr]
             )*
-            #[derive(Zeroize, ZeroizeOnDrop)]
+            #[derive(Zeroize, uniffi::Enum)]
```

Comment:
Have to skip `ZeroizeOnDrop` due to https://github.com/mozilla/uniffi-rs/issues/2080

I will give this some more thought, we might wanna ensure we pass secrets by mutable reference, so that we can call `.zeroize()` on them.